integration-authorization/src/main/java/org/symphonyoss/integration/authorization/oauth/v1/OAuth1Provider.java

```diff
@@ -77,6 +77,8 @@ public String requestTemporaryToken() throws OAuth1Exception {
         getAuthorizationCallbackUrl());
     try {
       return temporaryToken.getValue();
+    } catch (HttpResponseException e) {
+      throw new OAuth1HttpRequestException(e.getMessage(), e.getStatusCode());
```

Comment:
You might handle it as the bellow error handling. Using logMessage.getMessage().

Reply:
Which message should I put here once it's a general HTTP error?

Reply:
You are right. This will add more complexity to this. LGTM.